### PR TITLE
sbt 1.1.1

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ jdk:
 before_script:
   - sudo chmod +x /usr/local/bin/sbt
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION jvm/test
+  - sbt "++${TRAVIS_SCALA_VERSION}!" jvm/test
```
Comment (Rogach) [inline .travis.yml:11]: What does `!` mean in this case?

**Comment by Rogach** [comment]
Rejecting due to no response. If you will return to issue, feel free to reopen the PR.

File: .travis.yml (hunk 1 above)
Comment (xuwei-k) [reply .travis.yml:11]: force set scalaVersion. sbt 1.x changed `++` semantics. see https://github.com/sbt/sbt/pull/2613 https://github.com/sbt/website/issues/620

Comment (Rogach) [reply .travis.yml:11]: Thanks!

**Pull request merged**